app/src/main/java/com/android/sample/ui/createRecipe/PublishRecipeScreen.kt

```diff
@@ -0,0 +1,110 @@
+package com.android.sample.ui.createRecipe
+
+import android.widget.Toast
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.R
+import com.android.sample.model.recipe.CreateRecipeViewModel
+import com.android.sample.resources.C.Tag.CHEF_IMAGE_DESCRIPTION
+import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.navigation.Route
+import com.android.sample.ui.navigation.Screen
+import com.android.sample.ui.theme.Typography
+import com.android.sample.ui.utils.PlateSwipeScaffold
+
+@Composable
+fun PublishRecipeScreen(
+    navigationActions: NavigationActions,
+    createRecipeViewModel: CreateRecipeViewModel,
+) {
+  PlateSwipeScaffold(
+      navigationActions = navigationActions,
+      selectedItem = Route.CREATE_RECIPE,
+      showBackArrow = true,
+      content = { paddingValues ->
+        PublishRecipeContent(
+            navigationActions,
+            createRecipeViewModel,
+            modifier = Modifier.padding(paddingValues).fillMaxSize())
+      })
+}
+
+/**
+ * Composable function that displays the content for publishing a recipe.
+ *
+ * @param navigationActions Actions for navigating between screens.
+ * @param createRecipeViewModel ViewModel for managing the recipe creation process.
+ * @param modifier Modifier to be applied to the content.
+ */
+@Composable
+fun PublishRecipeContent(
+    navigationActions: NavigationActions,
+    createRecipeViewModel: CreateRecipeViewModel,
+    modifier: Modifier = Modifier
+) {
+  // Get the current context
+  val context = LocalContext.current
+
+  // Collect the publish error state
+  val publishError = createRecipeViewModel.publishError.collectAsState(initial = null).value
+
+  // Show a toast message if there is a publish error
+  publishError?.let {
+    Toast.makeText(context, it, Toast.LENGTH_SHORT).show()
+    createRecipeViewModel.clearPublishError()
+  }
+
+  Column(
+      modifier = modifier,
+      horizontalAlignment = Alignment.CenterHorizontally,
+      verticalArrangement = Arrangement.SpaceBetween) {
+        Spacer(modifier = Modifier.weight(0.05f))
+        // Display the done text
+        Text(
+            text = stringResource(R.string.done_text),
+            style = Typography.titleLarge.copy(fontSize = 70.sp, fontWeight = FontWeight.Bold),
```

Comment:
Just curious, why are you using `.copy` here? If there’s no existing typography style that suits this, maybe it would be clearer to create a new one in `Type.kt` for consistency and reusability across the app. Let me know if I can help with any details!